network/iptables.go

```diff
@@ -134,7 +145,7 @@ func (i *iptables) Intercept(protocol string, port string, cgroupPath string, in
 	// inject prerouting rule only if there's no cgroup path filtering
 	// packets going through prerouting chain are not yet associated
 	// to a process so there's no possibility to filter on a cgroup at this stage
-	if cgroupPath == "" {
+	if cgroupPath == "" && cgroupClassID == "" {
```

Comment:
Do you think a debug log here is pertinent?

Reply:
The insert call will already log at the info level the injected iptables command here so probably not useful to add one more thing.